front/public/locales/fr/errors.json

```diff
@@ -33,6 +33,10 @@
       "ConnectionResetByPeer": "Core: réinitialisation de la connexion. Nouvelle tentative.",
       "CoreResponseFormatError": "Core: impossible d'analyser la réponse '{{msg}}'",
       "GenericCoreError": "Core: erreur {{raw_error}}",
+      "Lapin": "Core: file d'attente de message: erreur de protocole",
```

Comment:
Why lapin haha ? :rabbit:

Reply:
Check my previous comment :D

Reply:
It's the rust library name to handle RabbitMQ protocole. :fr: :rabbit:  https://github.com/amqp-rs/lapin